test/ansi-to-html-tests.js

```diff
@@ -22,43 +22,44 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-import Filter from '../static/ansi-to-html';
-
-describe('ansi-to-html', () => {
-    const filterOpts = {
-        fg: '#333333',
-        bg: '#f5f5f5',
-        stream: true,
-        escapeXML: true,
-    };
-    it('Should leave non-ansi colours alone', () => {
-        const filter = new Filter(filterOpts);
-        filter.toHtml('I am a boring old string')
-            .should.equal('I am a boring old string');
-    });
-    it('Should handle simple cases', () => {
-        const filter = new Filter(filterOpts);
-        filter.toHtml('\x1B[38;5;99mTest')
-            .should.equal('<span style="color:#875fff">Test</span>');
-    });
-    it('Should handle nasty edge cases', () => {
-        const filter = new Filter(filterOpts);
-        // See #1666, this used to cause catastrophic backtracking.
-        filter.toHtml('\x1B[38;5;9999999999999999999999999999999999999999999999999999999999999999999999999999999' +
-            '99999999999999999999"mTest').should.equal(
-            '5;9999999999999999999999999999999999999999999999999999999999999' +
-            '99999999999999999999999999999999999999"mTest');
-    });
-
-    // With thanks to https://github.com/rburns/ansi-to-html/pull/84/files
-    it('renders xterm foreground 256 sequences', () => {
-        const filter = new Filter(filterOpts);
-        filter.toHtml('\x1B[38;5;196mhello')
-            .should.equal('<span style="color:#ff0000">hello</span>');
-    });
-    it('renders xterm background 256 sequences', () => {
-        const filter = new Filter(filterOpts);
-        filter.toHtml('\x1B[48;5;196mhello')
-            .should.equal('<span style="background-color:#ff0000">hello</span>');
-    });
-});
+// TODO(supergrecko): Re-enable these once TS test infrastructure is up
```

Comment:
We need TypeScript test infrastructure.

Reply:
That's not a blocker, though it would make me feel happier, it's just I wanted to make a note so I could "make an issue from comment"